game/Content/Audio/AudioConfig.yaml

```diff
@@ -25,6 +25,14 @@
         - "Music/AmbientMusic.3.m4a"
         - "Music/AmbientMusic.4.m4a"
         - "Music/AmbientMusic.5.m4a"
+        - "Music/sd2-1.m4a"
+        - "Music/sd2-2.m4a"
+        - "Music/sd2-3.m4a"
+        - "Music/sd2-4.m4a"
+    - SoundEffect:
+      Id: ShipyardTheme
+      Volume: 0.5
```

Comment:
0.5 is too low compared to the other music. I think 1.0 is just fine (tested it)>